src/stan/services/sample/standalone_gqs.hpp

```diff
@@ -84,6 +84,100 @@ int standalone_generate(const Model &model, const Eigen::MatrixXd &draws,
   return error_codes::OK;
 }
 
+/**
+ * Given a set of draws from a fitted model, generate corresponding
+ * quantities of interest which are written to callback writer.
+ * Matrix of draws consists of one row per draw, one column per parameter.
+ * Draws are processed one row at a time.
+ * Return code indicates success or type of error.
+ *
+ * @tparam Model model class
+ * @tparam SampleWriter type of sample writer
+ * @param[in] model instantiated model
+ * @param[in] num_chains number of chains
+ * @param[in] draws standard vector containing sequence of draws of constrained
+ * parameters
+ * @param[in] seed seed to use for randomization
+ * @param[in, out] interrupt called every iteration
+ * @param[in, out] logger logger to which to write warning and error messages
+ * @param[in, out] sample_writers A vector of writers to which draws for each
+ * chain are written
+ * @return error code
+ */
+template <typename Model, typename SampleWriter>
+int standalone_generate(const Model &model, const int num_chains,
+                        const std::vector<Eigen::MatrixXd> &draws,
+                        unsigned int seed, callbacks::interrupt &interrupt,
+                        callbacks::logger &logger,
+                        SampleWriter &sample_writers) {
```

Comment:
Seems odd to me that this argument isn't a std::vector?

I suppose the template is just expanded to be a std::vector of the writer, but for consistency I think it would be nice if it was the same as the other functions